docs/reference/run.md

```diff
@@ -345,7 +345,7 @@ Supported networks :
       <td class="no-wrap"><strong>container</strong>:&lt;name|id&gt;</td>
       <td>
         Use the network stack of another container, specified via
-        its *name* or *id*.
```

Comment:
![img_20161129_195516](https://cloud.githubusercontent.com/assets/9248427/20707224/df88c05c-b66d-11e6-9b12-cf32b608ce18.png)

Please fix this format as well?